koku/api/provider/models.py

```diff
@@ -189,6 +189,8 @@ class Meta:
     customer = models.ForeignKey("Customer", null=True, on_delete=models.PROTECT)
     created_by = models.ForeignKey("User", null=True, on_delete=models.SET_NULL)
     setup_complete = models.BooleanField(default=False)
+    # Used for data validation checks
+    data_valid = models.BooleanField(null=True, default=None)
```

Comment:
Along side this we _could_ store a json to store similar to what is logged during a failure. List of invalid days and their difference or something. It could aid investigations but it needs thought around multiple months, regarding if we just append to this json or replace it for example.

Reply:
Probs worth talking about all of these questions in a design discussion since it feels unclear what the outcome we are looking for is.

Reply:
Dropping this for the first iteration, we may want this when/if we add retry logic.